.github/FUNDING.yml

```diff
@@ -0,0 +1 @@
+custom: https://replit.com/JaydenDev
```

Comment:
We dont need this file

Reply:
Okay.

Reply:
Fixed. See https://github.com/hello-smile6/freecat/commit/8ffd77122c3ee4e4033547ad63033d527ae829f3